miri-script/src/commands.rs

```diff
@@ -227,8 +225,17 @@ impl MiriRunner<'_> {
         .run()?;
         Ok(())
     }
+}
+
+impl MiriRunner<'_> {
     fn bench(&self, benches: &[String]) -> Result<()> {
         // Make sure we have an up-to-date Miri installed
+        let hyperfine = std::env::var("HYPERFINE");
+        let hyperfine = hyperfine.as_deref().unwrap_or("hyperfine -w 1 -m 5 --shell=none");
```

Comment:
This is now verbose in a different direction but it would be _very cool_ if we could write shell commands like this in just one string literal.

Even cooler if we could stop shelling out all the time, but you take what you can get.

Reply:
(you reacted with confused so I'm going to try to clarify)

I find the vertical arrangement of these short CLI arguments to be a bit hard to read. The one for rustup-toolchain-install-master is a bunch of very short arguments each on their own line, and between all the multiple lines and all the string literal delimiters that's just not what it looks like in my terminal when I run rustup-toolchain-install-master. It would be very neat if we could have Rust code invoking CLI tools in a way that looks similar to when I run them by hand. I didn't even think of what you're doing here as a way to make that happen. It's interesting.

What would be _really_ nice is if we could do all of the tasks this program needs to do without spawning subprocesses, but that's probably just a dream, and just migrating this to Rust is a huge benefit in itself.

Reply:
Hey, sorry for the confusion; I didn't actually notice that it's a confused emoji - I thought it's a sad one, as I also find it a bit too verbose for my liking. I appreciate that you've reacted to it though =)
Emojis aside, it kinda sucks that we need to worry about knowing which part of `hyperfine` we should interpret as a command and which of them are actually parameters.

Reply:
Seems like [xshell](https://github.com/matklad/xshell) is just what we wanted. :)